src/main/resources/eu/markov/jenkins/plugin/mvnmeta/MavenMetadataParameterDefinition/help-currentArtifactInfoLabel.html

```diff
@@ -0,0 +1,29 @@
+<!--
+  - The MIT License
+  -
+  - Copyright (c) 2012, AKQA, Georgi "Gesh" Markov
```

Comment:
Yeah, most of my code was written while I was employed by AKQA as part of a project I was working there, so I had to add this for legal reasons, even if it is licensed freely.

You can certainly drop the copyrights or adapt them to your legal situation. As long as you maintain the MIT License it's all good I think.